src/SharpRaven.sln

```diff
@@ -81,96 +97,136 @@ Global
+		{156621FC-2C48-4CDF-A368-9347BABE9089}.Debug 4.5|Mixed Platforms.ActiveCfg = Debug 4.5|Any CPU
+		{156621FC-2C48-4CDF-A368-9347BABE9089}.Debug 4.5|Mixed Platforms.Build.0 = Debug 4.5|Any CPU
+		{156621FC-2C48-4CDF-A368-9347BABE9089}.Debug 4.5|x86.ActiveCfg = Debug 4.5|Any CPU
+		{156621FC-2C48-4CDF-A368-9347BABE9089}.Debug 4.5|x86.Build.0 = Debug 4.5|Any CPU
+		{156621FC-2C48-4CDF-A368-9347BABE9089}.Release 3.5|Any CPU.ActiveCfg = Release 3.5|Any CPU
+		{156621FC-2C48-4CDF-A368-9347BABE9089}.Release 3.5|Any CPU.Build.0 = Release 3.5|Any CPU
+		{156621FC-2C48-4CDF-A368-9347BABE9089}.Release 3.5|Mixed Platforms.ActiveCfg = Release 3.5|Any CPU
+		{156621FC-2C48-4CDF-A368-9347BABE9089}.Release 3.5|Mixed Platforms.Build.0 = Release 3.5|Any CPU
+		{156621FC-2C48-4CDF-A368-9347BABE9089}.Release 3.5|x86.ActiveCfg = Release 3.5|Any CPU
+		{156621FC-2C48-4CDF-A368-9347BABE9089}.Release 3.5|x86.Build.0 = Release 3.5|Any CPU
+		{156621FC-2C48-4CDF-A368-9347BABE9089}.Release 4.0|Any CPU.ActiveCfg = Release 4.0|Any CPU
+		{156621FC-2C48-4CDF-A368-9347BABE9089}.Release 4.0|Any CPU.Build.0 = Release 4.0|Any CPU
+		{156621FC-2C48-4CDF-A368-9347BABE9089}.Release 4.0|Mixed Platforms.ActiveCfg = Release 4.0|Any CPU
+		{156621FC-2C48-4CDF-A368-9347BABE9089}.Release 4.0|Mixed Platforms.Build.0 = Release 4.0|Any CPU
+		{156621FC-2C48-4CDF-A368-9347BABE9089}.Release 4.0|x86.ActiveCfg = Release 4.0|Any CPU
+		{156621FC-2C48-4CDF-A368-9347BABE9089}.Release 4.0|x86.Build.0 = Release 4.0|Any CPU
+		{156621FC-2C48-4CDF-A368-9347BABE9089}.Release 4.5|Any CPU.ActiveCfg = Release 4.5|Any CPU
+		{156621FC-2C48-4CDF-A368-9347BABE9089}.Release 4.5|Any CPU.Build.0 = Release 4.5|Any CPU
+		{156621FC-2C48-4CDF-A368-9347BABE9089}.Release 4.5|Mixed Platforms.ActiveCfg = Release 4.5|Any CPU
+		{156621FC-2C48-4CDF-A368-9347BABE9089}.Release 4.5|Mixed Platforms.Build.0 = Release 4.5|Any CPU
+		{156621FC-2C48-4CDF-A368-9347BABE9089}.Release 4.5|x86.ActiveCfg = Release 4.5|Any CPU
+		{156621FC-2C48-4CDF-A368-9347BABE9089}.Release 4.5|x86.Build.0 = Release 4.5|Any CPU
 	EndGlobalSection
 	GlobalSection(SolutionProperties) = preSolution
 		HideSolutionNode = FALSE
 	EndGlobalSection
 	GlobalSection(NestedProjects) = preSolution
 		{CC80A2E1-AE39-44DE-8DA3-4EEF42F90FB1} = {88CB5CD6-ACFA-441E-9FBB-E1E06F181B89}
-		{E1DBEBBF-9448-4D99-B378-2B8CF1629F31} = {D2B3990A-A048-49F3-A697-A1DAEF82CE7F}
-		{156621FC-2C48-4CDF-A368-9347BABE9089} = {D2B3990A-A048-49F3-A697-A1DAEF82CE7F}
+		{F44F1C47-DCF1-472A-A2D1-7F132AA54339} = {BF6154D1-9BC4-4966-9506-BC33E81F317C}
 		{ABE22746-6EEB-4970-A608-C02BC3B8BDA3} = {88CB5CD6-ACFA-441E-9FBB-E1E06F181B89}
 		{7FE52A2B-430C-4C6F-BEA9-0855AF973D0C} = {D2B3990A-A048-49F3-A697-A1DAEF82CE7F}
-		{F44F1C47-DCF1-472A-A2D1-7F132AA54339} = {BF6154D1-9BC4-4966-9506-BC33E81F317C}
+		{E1DBEBBF-9448-4D99-B378-2B8CF1629F31} = {D2B3990A-A048-49F3-A697-A1DAEF82CE7F}
+		{156621FC-2C48-4CDF-A368-9347BABE9089} = {D2B3990A-A048-49F3-A697-A1DAEF82CE7F}
```

Comment:
I am guessing Visual Studio? I am editing the solution file in Notepad++ while I have it opened in Visual Studio.